index.html

```diff
@@ -0,0 +1,72 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <title>Apple</title>
+  <meta charset="UTF-8" />
+  <link rel="stylesheet" href="styles.css" />
+  <script src="https://use.fontawesome.com/729dceb01f.js"></script>
+</head>
+
+<body>
+  <main>
+    <nav>
+      <div class="nav-first">
+          <i class="fa fa-apple"> <a href=""></a></i>
+          <i><a href="#">Store</a></i>
+          <i><a href="#">Mac</a></i>
+          <i><a href="#">iPod</a></i>
+          <i><a href="#">iPhone</a></i>
+          <i><a href="#">iPad</a></i>
+          <i><a href="#">iTunes</a></i>
+          <i><a href="#">Support</a></i>
+          <i class="fa fa-search"><a href=""></a></i>
```

Comment:
**[REQUIRED]** This should have a search field. Your project should have a [search field](https://www.w3schools.com/howto/howto_css_searchbar.asp) on the rightmost section.